### DISCUSSION TITLE
刻板印象何时消 | /posts/stereotypical/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#598] [Announcements]
https://yuanfan.rbind.io/posts/stereotypical/

**Comment by botsp** [comment]
那些刻板印象算作是某种程度的社会学规律吧，只是结论落于粗鄙了。
"小时偷针、长大偷金" 确实是要警惕小孩子偷拿偷占的习惯，养成良好的心理品格。当然即使小时偷针也不能意味着大时偷金。

感觉这个是很有意思的问题，俺以前也为此思索过。如果用这种刻板印象理解陌生世界、与陌生人打交道是有效的“捷径”，但用来理解身边人则失于浅薄。毕竟人是复杂的，值得好好阅读和理解。

PS.对于”4.斗争“，深有同感，这个世界实在有太多借着刻板印象进行恶意攻击的了，烦死啦。

**Reply by earfanfan** [reply]
>"小时偷针、长大偷金" 确实是要警惕小孩子偷拿偷占的习惯，养成良好的心理品格。当然即使小时偷针也不能意味着大时偷金。

其实我觉得这个点真得算是太站在大人的经验角度看问题了，如果真得有个小孩偷了一根针，应该是去耐心地跟小孩沟通为什么想要偷一根针，把那根针放回去可不可以，小孩子都不知道长大是个什么概念，不该用长大来威胁小孩，更不该用这种定论让尚且无法分辨好坏的小孩相信自己小时候偷了针就是品行很坏。（ps我在思考这个问题时，想到的小孩是我的小侄女，她才4岁）。

由于历史原因，我感觉我们国家的人民其实很擅长斗争，以及在斗争过程中使用划分阵营的方式拉帮结派地攻击对方。这一点如果深挖的话，涉及很多很多社会结构、经济发展、二次收入分配不公等宏观上的大问题。但表现在个人上的变化趋势就是，人人只考虑自己，所以才可以做到那么心安理得、肆无忌惮地攻击别人。

**Reply by yihui** [reply]
心理学书上经常讲小孩的偷窃行为或其它“犯罪”行为其实并不是为了偷那个东西，本质上是没有得到大人足够的关注，所以想用这些行为来吸引大人的注意力。从这个角度来说，小偷小摸并不是小孩的心理品格出了问题，而是大人有问题。

**Reply by botsp** [reply]
@earfanfan 确实，哈哈，正向引导比消极恐吓、实在益处多多。不止是小孩子，俺作为一个成年人现在也经常以这样的思路给自己心理按摩。　　　　


@yihui 感觉这个思路很西式，能平等尊重小孩子。

哭了、总是被指责是个爱哭的孩子；成绩差、就被指责懒惰不开窍。实在不公平，我是个小孩子又没办法和他平等辩论、探究原因。
成年人确实比孩子更有责任进行自省。

不过在"偷针"的问题上，是否要区分下年龄段，可能那种十几岁的“坏小子”需要迫切的教会他们什么是正确的社会行为规范，他们做出坏的行为可能只是基于顽皮的本性？
虽然没有十岁的娃，不过可以稍稍归纳下作为男性青年的成长经验，哈哈哈。

**Reply by earfanfan** [reply]
我觉得咱们几个的讨论就像我文中所写的那样，是不需分输赢的，而只是从各自不同的角度来探讨。其实我觉得@yihui提到的这点，哈哈，真得很中式呀，就是那种典型的“子不教，父之过”。在我的想象里，有丰富当爹经验的益辉，在育儿方面理所应当会更注重反省自己，也会更认同这类观点。

不过我上周末回去，所见却产生了一些疑问。我的小侄女才刚4岁，可是她竟然会欺负人了，我觉得这点可能和大人没关系。事情是酱紫的，我的小侄女有两个朋友，一个是女孩雅薇，另一个是男孩思雨，有一次男孩思雨把自己的玩具带到小侄女家来玩，他把玩具恐龙们摆好，此时雅薇和小侄女正在一起愉快玩耍没有理思雨，突然雅薇对小侄女说我们去搞破坏吧，于是她俩就一起把思雨的玩具们踢飞了而且以此为乐，完全不管思雨就站在旁边超大声哭泣……后来也不知道雅薇跟小侄女在说撒，思雨哭着跑回去了再来的时候，小侄女居然还去踢他……总之就是三个小孩一起玩就会有矛盾。如果我从大人视角来批判这些行为的话，肯定会很容易往道德层面上去想，可是对那几个4岁小孩来说，却撒也不是。因为这种短暂的不愉快，小孩子们居然很快就忘记。当雅薇被她妈带出门以后，小侄女还是会跑去找思雨玩。

作为大人，已经无意识接受了大量的社会规训，各种礼仪上、道德上的，所以可能很容易从一些小事上搜集样本进行批判或者反省。可是对那几个四岁的小孩来说，尚且不会有那么多的束缚，在某一个时刻感到不开心就哭就闹，在某一个时刻讨厌那个朋友就揍他，到了下一个时刻又会去找朋友快乐玩耍。

大人们对小孩的教养职责，可以说是需要让小孩懂得礼义廉耻，或者让小孩懂得正直善良等等等等，但是说到底依然是要逐步培养这个小孩成为一个合格的大人，进入社会参与社会分工成为社会的一份子，在这个过程中，是需要让一个如白纸一般的小孩去适应大人的世界。所以我觉得，如果一个孩子偷了一根针，是不是就要上纲上线，又是大人做反省又是批评小孩子的。当然了，我说的是偶然偷了一根针的情况，@botsp 说的应该是经常性偷针的情况，这种情况我觉得很难处理，因为我从小被教成了一个很符合大人期许的乖孩子，不明白“坏小子们”背后所需要的是什么，不过如果若干年后我成为了别人的母亲，刚好有个十岁的小孩总是忍不住偷针，我会试着首先忍住不用大人的那种道德观念去批判他定性他或她的“坏”，试着耐心地沟通倾听一下，然后再决定下一步该做点撒。

**Reply by botsp** [reply]
有画面了，有时候咱们大人反而容易上纲上线, 特别是作为敏感性人格，情绪雷达时刻开机，哈哈。在小孩子眼里确实只是转眼就忘的"屁大点儿事儿”。

"坏小子“类似初中生吧，国外应该也有teenager这样的群体标签(Troubled Teen)。这个年龄段的不能说为非作歹、但做事既不了解基本规则也缺少尊重的同理心，还弄不清情绪的重量。我自己感觉是这样，来自偏远地区的成长经验，碰到挺多这种行为的，要是大个十岁肯定被抓派出所、大个二十岁身边肯定没朋友的，哈哈。现在的校园霸凌通常还是值得一个新闻头条的，但在多年以前，挺多霸凌事件的。

一般家庭氛围比较好的同学大多没这样的问题吧，或者是年龄大些就好了。升入高中之后这种事情就少很多很多了，毕竟大家都稍微懂点事了。

**Reply by botsp** [reply]
说回刻板印象，总的来说俺觉得可以平常心些，应该不算一个很消极的思考方式。
当然被别人用刻板印象审视，就挺不舒服的，通常我都是一忍了之，管它呢。只能默默在内心警醒自己心态要保持开放，别把自己弄狭窄了。

**Reply by earfanfan** [reply]
@botsp 你这“一忍了之”真是令人印象深刻啊，我实在是又忍不住想吐槽。

话说那位思雨小男孩的故事是有后续的。我很好奇为撒明明是同样才4岁的年纪，他被两个小女孩欺负了除了哭就是哭，完全不反抗一下。于是我问了我妈，听我妈讲，之前三个小孩一起玩会吵闹会打架，思雨小朋友不仅打赢了还让另外两个小朋友脸上挂彩了，于是思雨妈妈就跟思雨讲道理，说他是个男子汉，一个是妹妹（4岁）另一个是姐姐（4岁半），希望他能让着她们。以前我只知道我是因为什么理由被大人规训要忍让，现在才知道原来男孩也会因为各种原因被规训要忍让。哈哈，总而言之，就是任何年龄任何性别都会被一些理由给束缚住，大家全都要按照社会规训各归其位……

不知你有没有读书的习惯，如果有的话不妨看看《被讨厌的勇气》，最近我有三个朋友都说这本书值得看看，我还没看完，按照现在的一些理解，那就是自己的心态可以自己有选择地调整一下。如果被人用刻板印象审视时感到很不舒服，可以换个角度想，那是别人的问题。如果是可以正常沟通的亲人朋友，大可以把不舒服的感受说出来，大家交流分歧和差异，便都可以接触到不一样的新认知。如果是不能够正常沟通的情况，啊，就当做在收集样本吧，观察社会行为、社会思想的样本，如果利用这些样本来思考一下，也是不错的锻炼呢。大概就是如此这般，认真关注一下自己内心的真实感受，除了忍受和忽略以外，用一些方式处理一下情绪。

另外，其实我还稍有点好奇，一个男青年成长过程中所遭遇的刻板印象对待有哪些，我能联想到的也只有头发长、说话轻声细语、哭会被大肆嘲讽。

**Reply by botsp** [reply]
谢谢推荐~ 听着描述还是很能引起共鸣的，哈哈，俺应该去读一读。那种充满傲慢、偏见的审视，就当作是观察社会样本了。实在没有心力去计较太多，可能也是一种权衡吧，对我来说去针锋相对、情绪成本更高了。

最近一家日本料理店写上了"中国人与韩国人不准入内”，新闻很受关注，搁我真就是转头就走、只会心里默念他是笨蛋。但换
个角度，有人愿意与这种刻板印象对抗、也会使所有人受益了。

总的来说积极的行为确实比消极的方式更有价值。

如果从性别上来讲，感觉男性受到的困扰要比女性少多了，哈哈，被吐槽爱哭可能只是心理负担。但女性被审视，似乎都是真刀真枪，影响职业发展、家庭生活的，这种实在糟粕。其实现代社会的生产作业几乎都办公室内了，很多女性的能力、产出都令人望尘莫及并深刻钦佩。希望这种新时代新印象能取代旧的刻板印象吧。

突然想到一个关于刻板印象的小故事 (沉默男青年突然想说好多好多话、haaaaaaa:
多年前的大学时代，有天下午要坐公交车去家教，但是走到校门口发现一毛钱都没带。好像那时候有支付宝、但还没有推广公交车乘车码一类的支付方式。但又很赶时间，那个新校区真是又大又偏远，校门口已经是公交车终点站了，如果跑步回宿舍就会错过班车了。

于是凭借已经在大学厮混一年多的厚脸皮就上车了，想着到地方找同事借点现金就好，反正返程也还是这个车，到时候补一份车费吧。我这样期期艾艾的和司机大叔解释了一通，然后陷入一分钟尴尬的争辩，你小子怎么能这样就上车呢！想得美！

关键是这一分钟之后，司机大叔双手离开方向盘握住我的手、突然讲了句"你是河南人吧"，很和善的说来吧来吧，放我上车。司机的刻板印象真是识别够准的，哈哈。

在九十年代大批河南人南下广东务工，除去贫穷，那个年代又好打架斗殴等等，实在名声不佳，估计是令司机大叔印象深刻。

每每想到这个真是心情复杂，大叔并没有丢一句河南人、然后羞辱地赶我下车，而是基于他的刻板印象给了一个不知事的年轻人别样的温暖。

**Reply by earfanfan** [reply]
@botsp 
>如果从性别上来讲，感觉男性受到的困扰要比女性少多了，哈哈，被吐槽爱哭可能只是心理负担。但女性被审视，似乎都是真刀真枪，影响职业发展、家庭生活的，这种实在糟粕。其实现代社会的生产作业几乎都办公室内了，很多女性的能力、产出都令人望尘莫及并深刻钦佩。希望这种新时代新印象能取代旧的刻板印象吧。

其实这段文字我读了以后，有一些想吐槽的点。不过这次我不打算写出来，因为按照历史经验，只要我把性别议题稍微说得深一点或者更现实更残酷一点，很快就被怀疑憋着撒心思暗戳戳地要打拳了。哈哈，我之前受到的言语上的伤害还没完全好，现在还没办法做到可以放心大胆地跟一位男青年交流此类问题。所以只这一部分我想先跳过，其他部分俺还是可以继续真诚交流的说。

关于对河南人的刻板印象，其实我是近几年才听说的，这可能是因为我和我身边的人本来就接触不到河南人，但是关于湖北人的刻板印象倒是已经听过好多年了。有句说法是“天上九头鸟，地上湖北佬”，就是说湖北人都特别精明会算计，关于这种说法的起源，好像也是大约二三十年前的时候，湖北省仙桃市很多人组团骗人，我还听我弟讲过说零几年的时候我们那还会拉横幅“警惕仙桃骗子”，不知道是不是他记错了，上个月我去见我同学，她有个同事就是仙桃人，然而也会说尽管自己是仙桃人但也觉得仙桃人总骗人、要是仙桃人开的店也不会去买东西……另外，听她们聊天简直可以集齐各省的地图炮说法了，我印象比较深的一个是福建人爱借钱不还，这个说法的源头也是二三十年前的时候，很多福建人向银行贷款做生意不还之类的。其实说来说去，因地域歧视而产生的刻板印象都是有历史原因在内的。固守刻板印象不肯更新的，问题还是出在那些带有偏见的人自己身上。

关于河南人刻板印象，我有一次听单口相声还见识到一种反向运用刻板印象的例子。话说相声演员说自己是河南人，在上海工作的时候总是遭受地域歧视被怀疑会偷井盖、打人之类的，有一次坐地铁的时候，眼前的空座位被一位老奶奶用包占了，他认真讲道理老奶奶都不肯让，于是他说了一句“我是河南人”，那位老奶奶听了就果断把包抱在怀里把座位让了出来。

话又说回来，你的这个小故事讲得好精彩啊，我都能感知到当年那个年轻人内心的情绪波动，我觉得你的笔力很好哇，不知道你有没有个人博客或者公众号之类的，如果有的话可不可以告诉我，我可以去踩一踩留个脚印。

**Reply by botsp** [reply]
哈哈，以后可以多多拍砖，俺耳根子软，啥话都能听进去。

博客刚开张一个来月，还是犹犹豫豫、腼腼腆腆的阶段，只挤出了几篇日记，暂时当作笔记本了。 菜鸟期待交流~

https://screeningme.blogspot.com/

**Reply by earfanfan** [reply]
@botsp 
哈，我已经飞速跑去看了看，本来想着放到订阅软件里囤起来慢慢看的，但是。
嘿，愣是没找到订阅按钮，在哪里呢？

**Reply by botsp** [reply]
刚刚添加了Feed RSS订阅按钮，右上方，刷新下试试，现在应该可以啦

**Reply by earfanfan** [reply]
@botsp  你的订阅链接<https://screeningme.blogspot.com/feeds/posts/default>我没看明白，我只见过像我的博客订阅链接这种的<https://yuanfan.rbind.io/index.xml>，我的订阅软件(Ego reader)识别不出来你的，于是只好手动收藏到浏览器文件夹了，原谅俺见识浅薄。另外，你的留言系统我也没看明白怎么用，我刚留了个“test one”，但是只是留言了，我再试试怎么把留言人的名字也戳上去……
